README.md

```diff
@@ -138,6 +138,27 @@ After resolving the import paths you can import GOV.UK Frontend by using:
 @import "@govuk-frontend/button/button";
 ```
 
+#### Global Styles
+
+In GOV.UK Frontend we want to provide a convenient way to have correct styles applied to your links (`<a>`) and paragraphs (`<p>`) without having to set required GOV.UK Frontend CSS classes for each instance of an element.
+
+If you have downloaded the compiled minified CSS file from [the `dist/ `folder](../dist/css), global styles will be set.
+
+We have chosen 'Opted-in' as the default behaviour. Inclusion is controlled by `$govuk-global-styles` variable in `globals/settings` folder.
+
+If you're importing our Sass files we have provided a way to opt-out of our global styles in your application.
+
+You can set the `$govuk-global-styles` variable to `false` in your application Sass file.
+```
+// application.scss
+
+$govuk-global-styles: false;
+
+@import "govuk-frontend/all/all";
+```
+If you choose this option, you will need to either use GOV.UK Frontend defined classes on links (`<a>`) and paragraphs (`<p>`). If you are also including GOV.UK Template, links and paragraph styles will be inherited from that stylesheet.
```

Comment:
this says 'either' but then just gives one course of action?

Reply:
yeah, ta. fixed now